sys/ztimer/Makefile.dep

```diff
@@ -108,3 +108,27 @@ ifneq (,$(filter ztimer_msec,$(USEMODULE)))
     USEMODULE += ztimer_periph_timer
   endif
 endif
+
+ifneq (,$(filter ztimer_sec,$(USEMODULE)))
```

Comment:
@benpicco how would this affet your `rtc_rtt` concerns

Reply:
Anyway, this is already an issue in master... so this does not make the problem worse..

Reply:
I'll try to find a solution for this in a follow up